support-models/src/main/scala/com/gu/support/workers/states/ExecutionTypeDiscriminators.scala

```diff
@@ -7,9 +7,7 @@ object ExecutionTypeDiscriminators {
   val contribution = "Contribution"
   val supporterPlus = "SupporterPlus"
   val tierThree = "TierThree"
-  val digitalSubscriptionDirectPurchase = "DigitalSubscriptionDirectPurchase"
-  val digitalSubscriptionGiftPurchase = "DigitalSubscriptionGiftPurchase"
-  val digitalSubscriptionGiftRedemption = "DigitalSubscriptionGiftRedemption"
+  val digitalSubscription = "DigitalSubscription"
```

Comment:
if there is a `DigitalSubscriptionDirectPurchase` in progress when it's deployed, it would fail to deserialise if the discriminator is changed to `DigitalSubscription`
Not a showstopper, but worth avoiding merging it if we're pushing people to digi subs for any reason.